src/apps/common/submodules/portListing/portListing.js

```diff
@@ -96,9 +105,17 @@ define(function(require) {
 				parent,
 				container;
 
+			monster.ui.generateAppLayout(self, {
+				menus: [
+					{
+						tabs: self.portListingCheckProfile() ? subTabs : tabs
+					}
+				]
+			});
+
 			if (isMonsterApp) {
 				parent = args.parent;
-				container = args.container;
+				container = $('#common_app_container .app-content-wrapper');
```

Comment:
I think that container could change depending on the app where this component is rendered.